inst/cpp/sim.cpp

```diff
@@ -117,15 +120,14 @@ List sim_wrapper_cpp (NumericVector A, List design, List par, NumericVector iov_
         start = 0;
       }
     }
-    if(start == 0) { // make a separate call to sim_cpp to make sure observation variables are initialized and stored
```

Comment:
This was just clean up, not related to this issue. The extra ode call was not actually needed at `start==0`, it just needs to be called at least once _before_ the variables are copied to the collection objects.